src/main/java/com/powsybl/study/server/StudyController.java

```diff
@@ -0,0 +1,194 @@
+            @ApiResponse(code = 409, message = "The study already exist"),
+            @ApiResponse(code = 500, message = "The storage is down or a file with the same name already exists")})
+    public ResponseEntity<NetworkIds> createStudy(@PathVariable("studyName") String studyName,
+                                                  @RequestParam("caseFile") MultipartFile caseFile,
+                                                  @RequestParam("description") String description) throws IOException {
+
+        if (studyService.studyExists(studyName)) {
+            throw new ResponseStatusException(HttpStatus.CONFLICT, STUDY_ALREADY_EXISTS);
+        }
+
+        if (studyService.studyExists(caseFile.getOriginalFilename())) {
+            throw new ResponseStatusException(HttpStatus.CONFLICT, CASE_ALREADY_EXISTS);
+        }
+
+        NetworkIds networkIds;
+        try {
+            networkIds = studyService.createStudy(studyName, caseFile, description);
+        } catch (PowsyblException e) {
+            throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, e.getMessage());
+        }
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON_UTF8).body(networkIds);
+    }
+
+    @GetMapping(value = "/studies/{studyName}")
+    @ApiOperation(value = "get a study")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "The study information"),
+            @ApiResponse(code = 404, message = "The study doesn't exist")})
+    public ResponseEntity<Study> getStudy(@PathVariable("studyName") String studyName) {
+        Study study = studyService.getStudy(studyName);
+        if (study == null) {
+            return ResponseEntity.notFound().build();
+        }
+        return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON_UTF8).body(study);
+    }
+
+    @DeleteMapping(value = "/studies/{studyName}")
+    @ApiOperation(value = "delete the study")
+    @ApiResponse(code = 200, message = "Study deleted")
+    public ResponseEntity<Study> deleteStudy(@PathVariable("studyName") String studyName) {
+        studyService.deleteStudy(studyName);
+        return ResponseEntity.ok().build();
+    }
+
+    @GetMapping(value = "/cases")
+    @ApiOperation(value = "Get the case list")
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "The case list"),
+            @ApiResponse(code = 500, message = "The storage is down")})
+    public ResponseEntity<Map<String, String>> getCaseList() {
+        try {
+            Map<String, String> caseList = studyService.getCaseList();
+            return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON_UTF8).body(caseList);
+        } catch (PowsyblException e) {
+            throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, e.getMessage());
+        }
+    }
+
+    @GetMapping(value = "/svg/{networkUuid}/{voltageLevelId}")
```

Comment:
I think network UUID should never be exposed outside of this web service (like in the frontend). Front end should only known the study name and inside every controller method that need a network UUID a query to the cassandra study db should be done to get it (it should be very fast and not perceptible from the UI)